environments/icds-new/postgresql.yml

```diff
@@ -78,6 +78,12 @@ dbs:
       host: pgucr
       query_stats: True
       django_migrate: True  # the dashboard models are stored here
+    - name: airflow
```

Comment:
does this need a host specified?

Reply:
no. it should default to the main db host (i think that is specified at the top of this file).